### PR TITLE
OS: Add support for the Siduction OS

### PR DESCRIPTION
**Pull request by lexruee**
This pull-request adds support for the [Siduction](https://siduction.org) GNU/Linux Distribution.
It's not the prettiest ascii logo, but it does the job :smile:.

neofetch_siduction-final

### CODE REVIEW

File: neofetch [hunk 1]
```diff
@@ -78,6 +78,12 @@ get_distro() {
                     *) distro="Red Star OS $(awk -F'[^0-9*]' '$0=$2' /etc/redstar-release)"
                 esac
 
+            elif [[ -f "/etc/siduction-version" ]]; then
```
Comment (dylanaraps) [inline neofetch]: Is this block needed?

What does `cat /etc/os-release` display?

Comment (lexruee) [inline neofetch]: Without this block it does unfortunately not work :confused:.
Here is the contents of  the `/etc/os-release` file from Siduction:

```
PRETTY_NAME="Debian GNU/Linux 9 (stretch)"
NAME="Debian GNU/Linux"
VERSION="9 (stretch)"
ID=debian
HOME_URL="https://www.debian.org/"
SUPPORT_URL="https://www.debian.org/support"
BUG_REPORT_URL="https://bugs.debian.org/"
```
Please note that the contents of this file is misleading as Siduction is based on the Debian Sid/Unstable branch. That is probably also why Siduction uses the file `/etc/siduction-version` to declare that it's a derivative of Debian Sid. Overall, the Siduction developers told me on IRC that is the proper way to declare a Debian-based Distribution.

**Comment by konimex** [comment]
Also, something is odd with your resolution output. You might want to open up an issue.

**Comment by dylanaraps** [comment]
Thanks! :+1:

**Comment by lexruee** [comment]
@dylanaraps Thanks for the merge and sorry for the leading whitespaces^^.

**Comment by dylanaraps** [comment]
All good, no worries. :)

**Pull request merged**